### PR TITLE
fix : Upcoming/Ongoing issue

### LINKED ISSUE #1846: Bug in deciding upcoming for different time zones
**Actual Behaviour**

The current implementation of the feature doesn't take care of different time zones.

**Expected Behaviour**

The feature should take into account different time zones as well. 

**Would you like to work on the issue?**

Yes.

**Target Milestone to solve the issue**

Till which [milestone](https://github.com/fossasia/open-event-android/milestones) will you be fixing the issue? 
https://github.com/fossasia/open-event-android/milestone/35

### PR DESCRIPTION
**Pull request by Shailesh351**
Fixes #1846

Changes:
- Use default shared preference to get preferences from settings(Because settings preference uses default preference which is by default private)
- Add isUpcomingMethod

As we have added ThreeTenABP(https://github.com/fossasia/open-event-android/pull/1879)So All the Date conversions take care of TimeZone.

### CODE REVIEW

**Comment by Shailesh351** [comment]
@iamareebjamal @heysadboy Please review

File: android/app/src/main/java/org/fossasia/openevent/utils/SharedPreferencesUtil.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
     private static SharedPreferences.Editor editor;
 
     static {
-        sharedPreferences = OpenEventApp.getAppContext().getSharedPreferences(ConstantStrings.FOSS_PREFS, Context.MODE_PRIVATE);
+        sharedPreferences = PreferenceManager.getDefaultSharedPreferences(OpenEventApp.getAppContext());
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/utils/SharedPreferencesUtil.java:13]: Why not FOSS_PREFS?

Comment (Shailesh351) [reply android/app/src/main/java/org/fossasia/openevent/utils/SharedPreferencesUtil.java:13]: All the settings preferences are stored in the default preferences. If we use different preference than we will not able to get values from default. Problem I have faced was at https://github.com/fossasia/open-event-android/blob/development/android/app/src/main/java/org/fossasia/openevent/OpenEventApp.java#L111 It always returns `false` if we use other preference than default preference

**Review by iamareebjamal — APPROVED** [review]

**Review by scandindian — APPROVED** [review]
Works great!

**Pull request merged**